server/api/src/test/java/org/hawkular/apm/server/api/model/zipkin/SpanTest.java

```diff
@@ -55,4 +65,62 @@ public void testPartialUrl() throws MalformedURLException {
         assertEquals(url.getPath(), result.getPath());
     }
 
+    @Test
+    public void testSerialization() throws IOException, ClassNotFoundException {
+        BinaryAnnotation binaryAnnotation = new BinaryAnnotation();
+        binaryAnnotation.setKey("http.method");
+        binaryAnnotation.setValue("GET");
+        binaryAnnotation.setEndpoint(createEndpoint("bonjour", "127.0.0.2", 8090));
+        binaryAnnotation.setType(AnnotationType.STRING);
+
+        Span span = new Span(Arrays.asList(binaryAnnotation));
+        span.setAnnotations(serverAnnotations());
+
+        Span deserializedSPan = (Span)deserialize(serialize(span));
+
+        Assert.assertEquals(span.binaryAnnotationMapping(), deserializedSPan.binaryAnnotationMapping());
```

Comment:
Why have three assertEquals - wouldn't it be better to just have one checking 'span' against 'deserializedSPan'? (possibly with lowercase P :)